tests/test_regex_identifier.py

```diff
@@ -386,3 +389,28 @@ def test_arn4():
     r = regex_identifier.RegexIdentifier()
     res = r.check(["arn:aws:s3:::my_corporate_bucket/Development/*"])
     assert "ARN" in str(res)
+
+
+def test_unix_timestamp():
+    r = regex_identifier.RegexIdentifier()
+    ts_from_ymd = lambda ymd: int(datetime.strptime(ymd, '%Y-%m-%d').timestamp())
+
+    res = r.check([str(ts_from_ymd('2020-01-01'))])
```

Comment:
What's the point of calculating the UNIX timestamps in the tests? It would be simpler and save milliseconds of time if you hardcode them in. So instead do `res = r.check(["1623914861"])` as an example.

Reply:
I would argue that although tests are meant to be entirely repeatable, this has a higher chance of finding bugs (due to the test changing) than without?

Reply:
I implemennted timestamp calcullation from strings because it's much more human readable than random strings of digits. I don't think saving 1 ms from test runs is worth the readability loss.

Reply:
I would argue that it's less readable actually. You can identify that it's a UNIX timestamp by the test name. For example the Bitcoin address tests are also not readable if you don't know what a Bitcoin address looks like but you know from the test name that it's a Bitcoin address. Also all of our other tests are strings like `1623914861`

Reply:
sure, up to the maintainers I guess! Generally having only machine readable literals in your code is just a bad practise, maybe it's not that bad for tests. Lemme know if this should be changed.

Reply:
Yeah I think you should change it. When I first looked at it, I didn't know what it was. It makes it more complex than if you just made them into strings.